src/dosbox.cpp

```diff
@@ -821,12 +821,16 @@ void DOSBOX_Init()
 	pint->Set_help("The OPL waveform is now sampled at the mixer's playback rate to avoid\n"
 	               "resampling.");
 
+	pbool = secprop->Add_bool("cms", when_idle, true);
+	pbool->Set_help("Enable or disable the CMS for SoundBlaster 1, 1.5 and 2.");
+
 	const char* oplmodes[] = {"auto", "cms", "opl2", "dualopl2", "opl3", "opl3gold", "none", nullptr};
 	pstring = secprop->Add_string("oplmode", when_idle, "auto");
 	pstring->Set_values(oplmodes);
 	pstring->Set_help("Type of OPL emulation ('auto' by default).\n"
 	                  "On 'auto' the mode is determined by 'sbtype'.\n"
-	                  "All OPL modes are AdLib-compatible, except for 'cms'.");
+	                  "All OPL modes are AdLib-compatible, except for 'cms'.\n"
+	                  "The 'cms'-type is deprecated.");
```

Comment:
Please remove the text about `cms` and the deprecation warning (we don't call deprecations out explicitly in the config descriptions). So only this should remain:

`All OPL modes are AdLib-compatible.`